substance/document/container.js

```diff
@@ -136,6 +136,10 @@ Container.Prototype = function() {
     return this.listView.length;
   };
 
+  this.hasContent = function(panelType) {
```

Comment:
@DavidMorenoCortina, I'm wondering in what situation `panelType` is used here, because if I remove it and change the code below, it gives the same effect for eLife articles. Is it a hook used elsewhere in other Lens implementations?

Reply:
@gnott I used panelType in case other panel types are added and their content only have to be shown based on listView. At this moment that check is redundant because all panels are resource type so it could be simplified